src/modules/mavlink/CMakeLists.txt

```diff
@@ -31,7 +31,18 @@
 #
 ############################################################################
 
-px4_add_git_submodule(TARGET git_mavlink_v2 PATH "${PX4_SOURCE_DIR}/mavlink/include/mavlink/v2.0")
+px4_add_git_submodule(TARGET git_mavlink_v2 PATH "${PX4_SOURCE_DIR}/mavlink/mavlink")
+
+set(MAVLINK_DIALECT "development")
+set(MAVLINK_GIT_DIR "${PX4_SOURCE_DIR}/mavlink/mavlink")
+set(CLIBRARY_OUT_DIR "${PX4_SOURCE_DIR}/mavlink/include/mavlink/v2.0")
+
+add_custom_command(OUTPUT px4_mavlink_gen.stamp
+    COMMAND "${PX4_SOURCE_DIR}/Tools/mavlink_generate_dialect.sh" ${MAVLINK_DIALECT} "${MAVLINK_GIT_DIR}" "${CLIBRARY_OUT_DIR}" ${PYTHON_EXECUTABLE}
+    COMMAND ${CMAKE_COMMAND} -E touch px4_mavlink_gen.stamp
```

Comment:
For convenience I wonder if this should have actual dependencies on common.xml, etc?

Reply:
What do you mean?

Reply:
Updated.